src/syft/lib/torchvision/allowlist.py

```diff
@@ -18,9 +18,19 @@
 # TODO: Normalize properties only exists on the object not on the class?
 # allowlist["torchvision.transforms.Normalize.inplace"] = "syft.lib.python.Bool"
 # TODO: mean and std are actually tuples
-# allowlist["torchvision.transforms.Normalize.mean"] = "syft.lib.python.List"
-# allowlist["torchvision.transforms.Normalize.std"] = "syft.lib.python.List"
-
+allowlist["torchvision.transforms.Normalize.mean"] = "syft.lib.python.Int"
+allowlist["torchvision.transforms.Normalize.mean"] = "syft.lib.python.Float"
+allowlist["torchvision.transforms.Normalize.mean"] = "syft.lib.python.Tuple"
+allowlist["torchvision.transforms.Normalize.mean"] = "syft.lib.python.List"
+allowlist["torchvision.transforms.Normalize.mean"] = "syft.lib.tensor.IntegerTensor"
+allowlist["torchvision.transforms.Normalize.mean"] = "syft.lib.numpy.ndarray"
+
+allowlist["torchvision.transforms.Normalize.std"] = "syft.lib.python.Int"
+allowlist["torchvision.transforms.Normalize.std"] = "syft.lib.python.Float"
+allowlist["torchvision.transforms.Normalize.std"] = "syft.lib.python.Tuple"
+allowlist["torchvision.transforms.Normalize.std"] = "syft.lib.python.List"
+allowlist["torchvision.transforms.Normalize.std"] = "syft.lib.tensor.IntegerTensor"
+allowlist["torchvision.transforms.Normalize.std"] = "syft.lib.numpy.ndarray"
```

Comment:
Shouldn't these be UnionGenerator? example:
```
allowlist["torch.Tensor.max"] = UnionGenerator[
    "syft.lib.python.Bool",
    "syft.lib.python.Float",
    "syft.lib.python.Int",
    "torch.return_types.max",
]
```

Reply:
Is there any specific need for this? The error is 
`` mean is not an attribute of Normalize``

Reply:
also mean, std are not functions. they are attributes of normalize function

Reply:
allowlist return type is set to a single one.